core/src/components/select/test/item/select.e2e.ts

```diff
@@ -45,11 +45,13 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
       await page.setContent(
         `
         <div id="container" style="display: flex">
-          <ion-item>
-            <ion-select label="Fruit" value="apple">
-              <ion-select-option value="apple">Apple</ion-select-option>
-            </ion-select>
-          </ion-item>
+          <ion-list>
```

Comment:
Brandy and I discussed that it would be good to account for `ion-list` here too. Amanda had found an edge case where [Brandy's previous fix](https://github.com/ionic-team/ionic-framework/pull/28594) did not work, so we wanted to test for that here.